charts/newrelic-infrastructure/values.yaml

```diff
@@ -113,13 +113,17 @@ kubelet:
     requests:
       cpu: 100m
       memory: 150M
+  # -- Max number of scraper rerun after scraper runtime error happens
+  scraperMaxReruns: 4
+  sink:
+    http:
+      # probeTimeout is the amount of time the main func to probe infra agent sidecar before giving up and restarting during scraper agent starting.
+      probeTimeout: 90s
```

Comment:
Should these be added to the chart README?

Reply:
Good point, I will add them to the readme